tasks/cycles/cycles.js

```diff
@@ -8,13 +8,39 @@
 let forFactorial, whileFactorial, doFactorial;
 const n = 10; //10! = 3628800
 
+forFactorial = 1;
+whileFactorial = 1;
+doFactorial = 1;
+let n1 = 1,
+  n2 = 1;
+
+for (let i = 1; i <= n; i++) {
+  forFactorial *= i;
+}
+
+while (n1 <= n) {
+  whileFactorial *= n1;
+  n1++;
+}
+
+do {
+  doFactorial *= n2;
+  n2++;
+} while (n2 <= 10)
+
+
 /** TODO
  * return concatenated string from an array of substring
  * assign result to variable str
  */
 let str;
 const substr = ["I", " love", " JS"];
 
+str = "";
+for (let i = 0; i < substr.length; i++) {
```

Comment:
JFYI

Also, your could use here the `for of`

Example: 
`
for (let el of substr) {
  str += el;
};
`